src/pages/DatasetCatalog.js

```diff
@@ -275,15 +297,11 @@ export default function DatasetCatalog(props) {
 
   const findPropertyValue = (dataSet, propName, defaultVal) => {
     const defaultValue = isNil(defaultVal) ? '' : defaultVal;
-    return span({}, [
-      getOr(defaultValue)('propertyValue')(find({ propertyName: propName })(dataSet.properties))
-    ]);
+    return getOr(defaultValue)('propertyValue')(find({ propertyName: propName })(dataSet.properties));
```

Comment:
Removed the span wrapping the element (so i can sort on the string value itself). This doesn't seem to have an effect on the rendering of the table